mods/ra/rules/structures.yaml

```diff
@@ -146,18 +146,42 @@ SPEN:
 		SpawnOffset: 0,-213,0
 		Facing: 96
 		ExitCell: -1,2
+		ProductionTypes: Submarine
 	Exit@2:
 		SpawnOffset: 0,-213,0
 		Facing: 160
 		ExitCell: 3,2
+		ProductionTypes: Submarine
 	Exit@3:
 		SpawnOffset: 0,0,0
 		Facing: 32
 		ExitCell: 0,0
+		ProductionTypes: Submarine
 	Exit@4:
 		SpawnOffset: 0,0,0
 		Facing: 224
 		ExitCell: 2,0
+		ProductionTypes: Submarine
+	Exit@b1:
+		SpawnOffset: -1024,1024,0
+		Facing: 160
+		ExitCell: 0,2
+		ProductionTypes: Ship
```

Comment:
I'm not able to get ships to produce from the sub pen at all, even after fixing `SelectExit` to return null when blocked.  Not sure what the problem is without digging deeper.

Reply:
You can't produce transports? Or you can't produce other ships? Because I thought subpens should only make submarines and transports.

Reply:
You're right, my bad.  I was confusing the behaviour with cheats enabled (primary subpens being able to produce ships) as a general feature.  But it isn't.